src/Serilog.AspNetCore/Serilog.AspNetCore.csproj

```diff
@@ -21,16 +22,30 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Serilog" Version="2.8.0" />
+    <None Include="images/serilog-extension-nuget.png" Pack="true" PackagePath="/" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Serilog" Version="2.9.0" />
     <PackageReference Include="Serilog.Extensions.Hosting" Version="3.0.0" />
     <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" />
-    <PackageReference Include="Serilog.Sinks.File" Version="4.0.0" />
+    <PackageReference Include="Serilog.Sinks.File" Version="4.1.0" />
     <PackageReference Include="Serilog.Sinks.Debug" Version="1.0.1" />
     <PackageReference Include="Serilog.Settings.Configuration" Version="3.1.0" />
-    <PackageReference Include="Serilog.Formatting.Compact" Version="1.0.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.0.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.0" />
+    <PackageReference Include="Serilog.Formatting.Compact" Version="1.1.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.1.*" />
```

Comment:
`*` wildcards get baked into the package as specific versions at the time of package build (not package install), and this has caused a bit of confusion historically. Because they make builds non-deterministic we have chosen to avoid them in other Serilog projects. Can we please use specific versions, here?

Reply:
i'm fine with both, wildcard, explicit latest or explicit 0 ?
both can be confusing to it's up to the maintainer to make a choice

as a very personal and opinionated, i find it useful to have this behavior so you just rebuild libs when needing for an explicit roll forward and having a release (even though being open on en right "should" be enough)
it make it easier to understand against which version it is as built => as you said the constraint on the package being installed are the only source of truth.

i encountered the opposite side effect ^^
building against 2.1.0 instead of 2.1.* confused user saying that it was built against potential bug fixed in later release ;)
i would then try to suggest "explicit latest" here

what's your call ?

will push the fix tomorrow regardless of 2.1.0 or 2.1.(can't remember the actual digit)

Reply:
Thanks for the follow-up :-)  - yes, the trade-off is an annoying one. For now I think being consistent with the rest of the Serilog projects is worthwhile.

I think your "explicit latest" suggestion is a good option, let's go with that 👍

Reply:
perfect I was opening nuget.org right now to anticipate that ^^

will be pushed in like ~5min

Reply:
done